### PR TITLE
docs: usage, privacy policy

### PR DESCRIPTION
**Pull request by aldy505**
along with updated help command and error logging

halo om. nanti kalo udah di proofread tolong langsung merge aja ya.

kalo males proofread dan mau langsung merge juga boleh kok wkwkkw.

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+- `/dukun` - Shows dukun leaderboard
+- `/dukun +<point>` - Increment someone's dukun point
+- `/dukun -<point>` - Decrement someone's dukun point
+
+<img src="../.github/images/dukun-leaderboard.jpg" width="300px" />
+<img src="../.github/images/dukun.jpg" width="300px" />
+
+## `/eval`
+
+Evaluate javascript expression, and shows the output. Bear in mind we limit the usage of a lots of stuff.
```
Comment (elianiva) [inline docs/USAGE.md:88]:
```suggestion
Evaluate javascript expression and shows the output. Bear in mind we limit the usage of a lot of stuff.
```

File: docs/USAGE.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+If you want to produce more powerful things, please create it on [JSBin](https://jsbin.com/),[JSFiddle](http://jsfiddle.net/),
+[Codepen](https://codepen.io/), [Codesandbox](https://codesandbox.io/), or even [Repl.it](https://repl.it/).
+
+- `/eval <javascript expression>`
+
+<img src="../.github/images/eval.jpg" width="300px" />
+
+## `/search`
+
+Shows you 10 results from a DuckDuckGo search. As simple as that. No ads. Filtered clean search results by the algorithm.
```
Comment (elianiva) [inline docs/USAGE.md:98]: ackshhually, 10 + BEST (defaultnya 2, ini yg punya preview)

**Review by elianiva — CHANGES_REQUESTED** [review]

File: docs/USAGE.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Will send you the definition of bliding ej.
+
+Available only on Teknologi Umum Group.
+
+- `/blidingej`
+- `/blidingej indo` or `/blidingej id` - Indonesian translation
+
+## `/quiz`
+
+Will send you a random quiz. Whether it'd be TOKI, general programming, or some tech-related trivia.
```
[the remark this answers is not in the document]
Comment (aldy505) [reply docs/USAGE.md:44]: lembaga yg nyelenggarain OSN

File: docs/PRIVACY_POLICY.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Bug fix, pin pointing where the error came from.
+`/dukun` command needs your ID and name to indicate who's who.
+That's it.
+
+## How do we protect your information?
+
+For error logging, we use a third party service called Sentry. They uses SSL connection to send data.
+You can read about [Sentry's privacy policy here](https://sentry.io/privacy/).
+
+For other cases, we use MongoDB Atlas for our NoSQL database needs. They also uses SSL connectoion.
```
[the remark this answers is not in the document]
Comment (aldy505) [reply docs/PRIVACY_POLICY.md:37]: ga. data disana ga persistent sifatnya. cuma cache doang. inmem kan juga ada.

File: docs/USAGE.md (hunk 2 above)
Comment (aldy505) [reply docs/USAGE.md:98]: well yeah but you wrote it as "10 result", so don't confuse people🤷🏻‍♂️

**Comment by aldy505** [comment]
ok no response. let's merge this.

**Pull request merged**